MagneticField/src/LUXEMagneticField.cpp

```diff
@@ -0,0 +1,56 @@
+#include "ActsLUXEPipeline/LUXEMagneticField.hpp"
+
+namespace LUXEMagneticField {
+
+BField_t buildLUXEBField() {
+    Acts::MagneticFieldContext mfContext = Acts::MagneticFieldContext();
+
+    // definition of BField
+    struct BField {
```

Comment:
Also, maybe we can keep the struct as an argument, so that the build function is mainly responsible for binning. I think that way it'll be cleaner, because we'll just define several `BFields` in the header and will give them to the general builder in `main`.